game/src/common/waypoints.rs

```diff
@@ -191,8 +188,8 @@ impl InputWaypoints {
                     let idx = x.parse::<usize>().unwrap();
                     self.waypoints.remove(idx);
                     // Recalculate labels, in case we deleted in the middle
```

Comment:
Now that each waypoint doesn't "know" its position, we actually don't need to do any of this!

Reply:
Removed the portion where we reassemble the waypoints and just skip to updating drawable.